packages/react-dom/src/__tests__/ReactDOMFloat-test.js

```diff
@@ -137,13 +137,22 @@ describe('ReactDOMFloat', () => {
     buffer = '';
   }
 
-  function getVisibleChildren(element) {
+  function getVisibleChildrenAndResources(element) {
     const children = [];
     let node = element.firstChild;
     while (node) {
       if (node.nodeType === 1) {
         if (
-          node.tagName !== 'SCRIPT' &&
+          // Only include scripts with src elements, this will hide instruction scripts
+          !(node.tagName === 'SCRIPT' && !node.hasAttribute('src')) &&
+          // Only include style tags with actual content if they have a data-rprec attribute
+          // We use this to hide the placeholders which are used to emit insertion points when
+          // a style is not used in the shell but the precedence is encoded there
+          !(
```

Comment:
Do we still need this after we go back to the old semantics of precedence?

Reply:
changed back